Source/FlowEditor/Private/DetailCustomizations/Pre_5_3_GetActorClassDefaultComponents.h

```diff
@@ -0,0 +1,82 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "Runtime/Launch/Resources/Version.h"
+
+// NOTE (gtaylor) Compatibility patch for pre-5.3 UE to support AActor::GetActorClassDefaultComponents()
+#define USE_PRE_5_3_GET_ACTOR_CLASS_DEFAULT_COMPONENTS (ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION < 3)
+
+#if USE_PRE_5_3_GET_ACTOR_CLASS_DEFAULT_COMPONENTS
+
+#include "Components/ActorComponent.h"
+#include "Engine/BlueprintGeneratedClass.h"
+#include "Engine/SCS_Node.h"
+#include "Engine/SimpleConstructionScript.h"
+#include "GameFramework/Actor.h"
+
+FORCEINLINE_DEBUGGABLE void Pre_5_3_ForEachComponentOfActorClassDefault(const TSubclassOf<AActor>& ActorClass, const TSubclassOf<UActorComponent>& InComponentClass, TFunctionRef<bool(const UActorComponent*)> InFunc)
```

Comment:
Strictly a copy/paste of AActor functions from 5.3 to add pre-5.3 compatibility for this function